desktop/mac/release-notes/index.md

```diff
@@ -35,6 +35,43 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 ### Security
 
+- CVE-2022-0847
+
+### New
+
+- Virtiofs released as an experimental feature
+- Added volumes UI bulk removal
+
+### Upgrades
+
+- [Compose v2.3.3](https://github.com/docker/compose/releases/tag/v2.3.3)
+- [containerd v1.4.13](https://github.com/containerd/containerd/releases/tag/v1.4.13)
+- [runc v1.0.3](https://github.com/opencontainers/runc/releases/tag/v1.0.3)
+- [Go 1.17.8](https://golang.org/doc/go1.17)
+- [Linux kernel 5.10.104](https://hub.docker.com/layers/docker/for-desktop-kernel/5.10.104-379cadd2e08e8b25f932380e9fdaab97755357b3/images/sha256-7753b60f4544e5c5eed629d12151a49c8a4b48d98b4fb30e4e65cecc20da484d?context=explore)
+
+### Bug fixes and minor changes
+
+- Improved the error message from `docker build` when using Registry Access Management.
+- Fixed uploading diagnostics when an HTTPS proxy is set.
+- Fixed the systray menu not displaying all menu items after starting Docker Desktop. Fixes [docker/for-mac#6192](https://github.com/docker/for-mac/issues/6192).
+- Fixed a regression about Docker Desktop not starting in background anymore. Fixes [docker/for-mac#6167](https://github.com/docker/for-mac/issues/6167).
+- Fixed Docker Desktop Dock icon missing. Fixes [docker/for-mac#6173](https://github.com/docker/for-mac/issues/6173).
```

Comment:
@trungutt @StefanScherer should this one be listed for Windows as well?

Reply:
Yes, it happened on Mac and Windows, (my comment above ☝️ ). I put here different link for each

- Fixed Docker Desktop Dock icon missing. Fixes [docker/for-mac#6173](https://github.com/docker/for-mac/issues/6173).
- Fixed Docker Desktop Dock icon missing. Fixes [docker/for-win#12573](https://github.com/docker/for-win/issues/12573).